### PR TITLE
fix documentation on add_table. Fixes 9355

### LINKED ISSUE #9355: Potentially incomplete documentation for ConvergenceTable add_value() example
In the [documentation page](https://dealii.org/developer/doxygen/deal.II/classConvergenceTable.html) for the `add_value()` method of the `ConvergenceTable` class, we have

>The number of cells and the number of DoFs may be added to the table by calling e.g. add_value("n cells", n_cells). 

Does `n_cells` already contain the number of dofs?  Also, shouldn't it be `triangulation.n_cells` if I remember correctly from some early tutorials?

### PR DESCRIPTION
**Pull request by krishnakumarg1984**
Fixes #9355

### CODE REVIEW

File: include/deal.II/base/convergence_table.h [hunk 1]
```diff
@@ -38,11 +38,12 @@ DEAL_II_NAMESPACE_OPEN
...
  *
- * The number of cells and the number of DoFs may be added to the table by
- * calling e.g.  <tt>add_value("n cells", n_cells)</tt>. The table data is
- * also added by calling add_value().  Before the output of the table the
- * functions evaluate_convergence_rates() and evaluate_all_convergence_rates()
- * may be called.
+ * The number of cells may be added to the table by
+ * calling e.g.  `add_value("n cells", n_cells)`. The number of DoFs may be
+ * added to the table by calling `add_value("n dofs", n_dofs)`. Further table
+ * data is also added by calling add_value().  Before the output of the table
```
Comment (bangerth) [inline include/deal.II/base/convergence_table.h]: Let's either remove this middle sentence or say something like "Of course, one can also add more kinds of information by calling add_value() with different arguments.

Comment (bangerth) [reply include/deal.II/base/convergence_table.h]: @krishnakumarg1984 -- any news?

**Comment by drwells** [comment]
@krishnakumarg1984 Do you have time to fix this so we can merge?

**Comment by krishnakumarg1984** [comment]
Not currently, but I do intend to get back to this sometime in the future.

**Comment by tjhei** [comment]
I think we should merge as is. If somebody feels strongly about the surrounding text (@bangerth ), please open another pr.

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
Fair point.

**Pull request merged**